resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/InjectorFactoryImpl.java

```diff
@@ -130,15 +130,15 @@ public ValueInjector createParameterExtractor(Parameter parameter, ResteasyProvi
 
 
    @Override
-   public ValueInjector createParameterExtractor(Class injectTargetClass, AccessibleObject injectTarget, Class type,
+   public ValueInjector createParameterExtractor(Class injectTargetClass, AccessibleObject injectTarget, String defaultName, Class type,

```

Comment:
@asoldano This is backward incompatible change, right? I'm looking to TS changes and it seems that this should affect user's deploments ... maybe, it's not necessary to change signature of old public method, we can add new method with additional defaultName parameter, and old method can just call new method with additional defaultName parameter with some reasonable default ...

What are the plans for 3.6 branch?